styles.css

```diff
@@ -94,4 +141,41 @@ ol {
   text-align: center;
   align-items: center;
   justify-content: center;
-}
\ No newline at end of file
+
+}
+img {
+  max-height: 100%;
+  min-width: 100%;
+  object-fit: cover;
+  vertical-align: bottom;
+}
+.container2 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+
+  
+  }
+  .box2{
```

Comment:
if this is a common container box I'd suggest giving it a more descriptive class name, i.e. something like `charactersBox`